task/01-strings-tasks.js

```diff
@@ -130,7 +150,11 @@ function repeatString(value, count) {
  *   'ABABAB','BA' => 'ABAB'
  */
 function removeFirstOccurrences(str, value) {
-    throw new Error('Not implemented');
+    let first_i=str.indexOf(value);
+    let first_str=str.substring(0, first_i);
+    let second_str=str.substring((first_i+value.length), str.length);
+    let new_str=first_str+second_str;
+    return new_str;
```

Comment:
We can use String.prototype.replace
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace